src/Command/MarkMigrated.php

```diff
@@ -150,4 +159,95 @@ protected function markAllMigrated($path)
         }
         $adapter->commitTransaction();
     }
+
+    /**
+     * Decides based on $input if it will try to mark more than one migration
+     *
+     * @param InputInterface $input
+     * @return boolean
+     */
+    protected function isBatchMarking(InputInterface $input)
+    {
+        $version = $input->getArgument('version');
+
+        return $this->isAllVersion($version) ||
+            $this->isSkipTo($input) ||
+            $this->isUpTo($input);
+    }
+
+    /**
+     * Decides based on $input which versions it should mark as migrated
+     *
+     * @param InputInterface $input
+     * @return array of versions that should be marked as migrated
+     */
+    protected function versionsToMark(InputInterface $input)
```

Comment:
After all you "get" something and method names should be as descriptive as possible while keeping them reasonably short and adhering to a given CS.